Fix Skia-to-SDL pipeline for any endianness (#294)
browserengineering/book: book/visual-effects.md

        # doesn't actually copy anything yet.
        skia_image = self.root_surface.makeImageSnapshot()
        skia_bytes = skia_image.tobytes()
```

Next, we need to copy the data to an SDL surface. This requires

> telling SDL what order the pixels are stored in and on your computer's

Reviewer: ```suggestion
telling SDL what order the pixels are stored in (that's why we specified RGBA_888 above).
```

> [endianness][wiki-endianness]:

Reviewer: ```suggestion
```

> [wiki-endianness]: https://en.wikipedia.org/wiki/Endianness

Reviewer: ```suggestion
```
